README.md

```diff
@@ -83,6 +83,11 @@ To execute the generator execute this command. By default it will look for the `
 - `--includeBlunder=False` If False then generated puzzles won't include initial blunder move, default is `True`
 - `--stockfish=./stockfish-x86_64-bmi2` Path to Stockfish binary. 
   Optional. If omitted, the program will try to locate Stockfish in current directory or download it from the net
+- `--color=WHITE` Generate puzzles only for preferred color. `WHITE`/`W`, `BLACK`/`B`, or default `NONE` 
```

Comment:
Can you include a test to make sure that the filters are working correctly? 
For example with this game.pgn file https://gist.github.com/vitogit/8e1059e57929a9fde4f69232eca6c1f2 
Running `python3 main.py --depth=5 --games=game.pgn` it generates 3 puzzles
Running `python3 main.py --depth=5 --games=game.pgn --color=WHITE` it generate 1 puzzle
Running `python3 main.py --depth=5 --games=game.pgn --color=BLACK` it generate 1 puzzle when it should to generate 2.